build.gradle

```diff
@@ -264,8 +264,8 @@ dependencies {
     compile 'org.jooq:jool-java-8:0.9.14'
     compile 'com.github.hmcts:ccd-case-document-am-client:1.7.1'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.14.0'
-    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.2-Categories'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.13.5-prerelease'
+    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.1'
```

Comment:
Would this now lose the specific delta in 8.7.2-Categories? I'd suggest releasing '8.7.2-Categories' into a next stable release and use that one, if I'm not getting it wrong.